### ISSUE TITLE
📝 Correction |  Recommended Browser Add-ons Android (Etag Stoppa and Cookie AutoDelete

### ISSUE THREAD

**Issue by ph00lt0** [privacytools/privacytools.io#2381]
On the website, currently Etag Stoppa and Cookie AutoDelete are recommended, however these add-ons are not supported for the current version of Firefox on Android.

Only the following add-ons are supported: https://addons.mozilla.org/en-US/android/

I have no formal connection with the software.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by ph00lt0** [comment]
We might want to add Privacy Possum (https://github.com/privacytools/privacytools.io/issues/459) as an recommendation, this add-on blocks Etag headers.

**Comment by Thorin-Oakenpants** [comment]
What does Etag Stoppa achieve given that FF85+ shipped with [network partitioning](https://blog.mozilla.org/security/2021/01/26/supercookie-protections/)
 - hint: please don't say repeat visits within a session, as this ignores many other tracking vectors: browsers simply are not designed to mask in-session repeat visits, *not even Tor Browser*. If this is you aim, then you're doing it wrong
 - hint: please don't say repeat visits across sessions (clear your cache on close: it's called sanitizing)

**Comment by Thorin-Oakenpants** [comment]
[Privacy Possum](https://addons.mozilla.org/firefox/addon/privacy-possum/) .. last updated over two years ago .. but wait, there's more ... if you look at the [repo](https://github.com/cowlicks/privacypossum) or at the [version history](https://addons.mozilla.org/en-US/firefox/addon/privacy-possum/versions/) it's **_almost three years_** since anything meaningful was done .. and that's not even getting started on the fingerprinting issues

Please don't suggest abandonware

While blocking ETAGs won't cause harm (it is fingerprintable, but not worth anyone's time or effort IMO), Clear URLS can provide this functionality and ... **_OMG_** ... it's already a [recommended addon](https://www.privacytools.io/browsers/#addons)

**Comment by dngray** [comment]
Regarding this,  it will likely be taken care of once we do https://github.com/privacytools/privacytools.io/pull/2081

> Clear URLS can provide this functionality

I'm kind of hoping https://gitlab.com/KevinRoebert/ClearUrls/-/issues/597 might mean that it will be supported soon.

**Comment by ph00lt0** [comment]
@Thorin-Oakenpants this is why I sad 'might', just trying to find a replacement, CleanURLs is not available on android btw and I am not sure who those hints are for!?
I am just providing a possible alternative for add-ons that can't be installed in the android client at all.

**Comment by ph00lt0** [comment]
@dngray thanks, that looks good indeed. Not sure why Umetrix is still recommended but that is for another topic.

**Comment by Thorin-Oakenpants** [comment]
> I am just providing a possible alternative for add-ons that can't be installed in the android client at all

your suggestions (Privacy Possum, Etag Stoppa) cannot be installed in android. You were clearly pushing in-session sanitizing, which means you are doing it wrong

**Comment by ph00lt0** [comment]
@Thorin-Oakenpants you make a lot of false assumptions and accusations. Privacy possum can be installed as add-on android, the current recommendation is Etag Stoppa (not submitted by me) can no longer be installed.

I am not going to discuss with you further, your toxic comments are not helping this community.

**Comment by rusty-snake** [comment]
> Only the following add-ons are supported: https://addons.mozilla.org/en-US/android/

![Screenshot 2021-07-20 at 16-59-20 Add-ons for Firefox Android (en-US)](https://user-images.githubusercontent.com/41237666/126365174-6ff05570-6495-4dfa-a606-4ba68d66173d.png)

**Comment by ph00lt0** [comment]
@rusty-snake click on "See more recommended extensions" 

Or directly:
https://addons.mozilla.org/en-US/android/search/?promoted=recommended&sort=random&type=extension

**Comment by youdontneedtoknow22** [comment]
I have no idea what "Etag Stoppa" does, but I wanted to contribute with something else.

ClearURLs isn't availabe on android, but uBO is. We probably need some more testing, but uBO can now replace ClearURLs with an Adguard List. And if it's done correctly then again: less cpu usage + less attack surface.
https://www.reddit.com/r/privacytoolsIO/comments/ooie4u/psa_ublock_origin_added_two_new_stock_filter/

**Comment by ph00lt0** [comment]
@youdontneedtoknow22 it blocks the tracking of these: https://en.wikipedia.org/wiki/HTTP_ETag Privacy Possum also does this. I am not sure if ublock origin does this as well.

**Comment by dngray** [comment]
> @rusty-snake click on "See more recommended extensions"

You know what is funny about that, I wouldn't install Ghostery, Decentraleyes, Privacy Possum or Privacy Badger. Decentraleyes and Privacy Possum are not actually updated and haven't been for years despite being available in the new Fenix builds.

Only addon I have on Android is uBlock Origin, and HTTPS Everywhere. I guess I could enable `dom.security.https_only_mode`, the UI doesn't seem exposed for that yet.

**Comment by ph00lt0** [comment]
@dngray I fully agree with you as long as `privacy.firstparty.isolation` is on. I think also from the conversation at #2081 we should try to confirm this. The only way I see to read the profile on the default Firefox installation on android is with a rooted device which I do not have.
I am not trying to push it on you, but just wanted to let you know that in Mull actually you are actually able to open about:config unlike in the default fenix.

Either way Etag Stoppa and Cookie AutoDelete shouldn't be listed on the website anymore.

**Comment by dngray** [comment]
This whole page needs to be re-done now, and it's pretty much changed in the time i started the PR till now.
